amf-client/shared/src/test/scala/amf/error/RamlParserErrorTest.scala

```diff
@@ -98,19 +98,15 @@ class RamlParserErrorTest extends ParserErrorTest {
   test("Inline external fragment from non mutable ref") {
     validate(
       "error/inline-non-mutable-ref/api.raml",
-      badInclude1 => {
-        badInclude1.level should be("Violation")
-        badInclude1.message should startWith("Fragments must be imported by using '!include'")
-      },
-      badInclude2 => {
```

Comment:
Why do we have one less assertion?

Reply:
This was due to the change of verifyValidReferenceKind. This test case has a an external fragment that references a raml library incorrectly. This incorrect reference used to be validated with the raml plugin (even though the file in not raml related), while now as these reference validation are specific for raml it is no longer present.
We analyzed this behavior change with @pope1838 and agreed that the impact is very reduced and can be considered correct.